doc/classes/Input.xml

```diff
@@ -204,6 +206,7 @@
 				Returns [code]true[/code] if you are pressing the action event. Note that if an action has multiple buttons assigned and more than one of them is pressed, releasing one button will release the action, even if some other button assigned to this action is still pressed.
 				If [param exact_match] is [code]false[/code], it ignores additional input modifiers for [InputEventKey] and [InputEventMouseButton] events, and the direction for [InputEventJoypadMotion] events.
 				[b]Note:[/b] Due to keyboard ghosting, [method is_action_pressed] may return [code]false[/code] even if one of the action's keys is pressed. See [url=$DOCS_URL/tutorials/inputs/input_examples.html#keyboard-events]Input examples[/url] in the documentation for more information.
+				[b]Note:[/b] During input-handling (e.g. [method Node._input]) use [method InputEvent.is_action_pressed] instead, to query the action-state of the current event.
```

Comment:
Unlike `is_action_just_*()`, I think it's OK to use `is_action_pressed()` here. It has a different semantics from checking `is_action_pressed()` of the event.

Reply:
Alright, I have reverted this statement. Thanks for pointing this out.